src/backend/api/Fusion.Resources.Domain/Commands/ResponsibilityMatrix/UpdateResponsibilityMatrix.cs

```diff
@@ -125,6 +111,18 @@ public async Task<QueryResponsibilityMatrix> Handle(UpdateResponsibilityMatrix r
                 var returnItem = await mediator.Send(new GetResponsibilityMatrixItem(request.Id));
                 return returnItem!;
             }
+
+            private async Task<DbPerson?> GetResourceOwner(string departmentId)
+            {
+                var department = await lineOrgResolver.GetDepartment(departmentId);
+                if(department?.Responsible?.AzureUniqueId is not null)
```

Comment:
I would expect a valid department to have a responsible person.
Or ?

Reply:
I'm not sure we can guarantee that with the current solution for departments; an admin can theoretically add a department that does not exist in line org. That is something I'm revising now, but is worth an entire new PR.